src/jit/codegenlegacy.cpp

```diff
@@ -18662,9 +18662,8 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
 #ifdef _TARGET_ARM_
         if ((call->gtCallMoreFlags & GTF_CALL_M_SECURE_DELEGATE_INV))
         {
-            getEmitter()->emitIns_R_R_I(INS_add, EA_PTRSIZE, compiler->virtualStubParamInfo->GetReg(), regThis,
+            getEmitter()->emitIns_R_R_I(INS_add, EA_BYREF, compiler->virtualStubParamInfo->GetReg(), regThis,
                                         pInfo->offsetOfSecureDelegateIndirectCell);
-            regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
         }
 #endif // _TARGET_ARM_
 
```

Comment:
The diffs as shown by Git are wrong here.

What is shown would be a compile time error

Reply:
Right, that's interesting. The github shows a nonsense here.
Git diff for that change shows this:
```diff
diff --git a/src/jit/codegenlegacy.cpp b/src/jit/codegenlegacy.cpp
index c39d17419f..18a2db3a02 100644
--- a/src/jit/codegenlegacy.cpp
+++ b/src/jit/codegenlegacy.cpp
@@ -18662,9 +18662,8 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
 #ifdef _TARGET_ARM_
         if ((call->gtCallMoreFlags & GTF_CALL_M_SECURE_DELEGATE_INV))
         {
-            getEmitter()->emitIns_R_R_I(INS_add, EA_PTRSIZE, compiler->virtualStubParamInfo->GetReg(), regThis,
+            getEmitter()->emitIns_R_R_I(INS_add, EA_BYREF, compiler->virtualStubParamInfo->GetReg(), regThis,
                                         pInfo->offsetOfSecureDelegateIndirectCell);
-            regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
         }
 #endif // _TARGET_ARM_

@@ -18689,6 +18688,13 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                    INDEBUG_LDISASM_COMMA(sigInfo) NULL, // addr
                                    args, retSize, gcInfo.gcVarPtrSetCur, gcInfo.gcRegGCrefSetCur,
                                    gcInfo.gcRegByrefSetCur, ilOffset, indCallReg);
+
+#ifdef _TARGET_ARM_
+        if ((call->gtCallMoreFlags & GTF_CALL_M_SECURE_DELEGATE_INV))
+        {
+            regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
+        }
+#endif // _TARGET_ARM_
     }
     else
```

Reply:
Also when I tried to view the whole file, it wouldn't go beyond line 16000 or so...

Reply:
That's weird, because what you show above is what I'm seeing on github.